src/pages/Products/ProductsCardList.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import ProductsCard from './ProductsCard';
+import styled from 'styled-components';
+
+class ProductCardList extends Component {
+  render() {
+    //nconsole.log(this.props);
+    return (
+      <Container>
+        {this.props.productsList.map(productsData => {
```

Comment:
map 메소드 잘 활용하셨습니다! :) 굿굿

아래와 같이 구조분해할당 해주시면 좋겠네요! 
더불어 productsList를 map 메소드를 사용할경우, 인자에 담기는 값이 복수가 아니라 단수이므로 
productData가 더 적합할 것 같습니다 :) 
```suggestion
  render() {
    const { productsList } = this.props;
    return (
      <Container>
        {productsList.map(product => {
```

Reply:
반영했습니다! productsData는 수정하는 과정에서 없앴지만, 추후 또 만들게 된다면 의미를 잘 담아 productData로 만들어 보겠습니다!